### PR TITLE
E3 Fake page, made only for academic reasons based on the canceled E3 video game conference. 

### PR DESCRIPTION
**Pull request by CarlosZ96**
The skeleton of the main section and main parts of the program in the mobile version is created, the main text is added, the hamburger menu is created, the main program section is finished. Background images, icons, font styles and colors are added, for the "main program" section the elements are created dynamically with HTML and JavaScript. The speakers section and the about section, icon section, past E3 section, footer are dynamically added, part of the content is hidden with JavaScript to show the "about" page when clicking see more, and it is hidden when clicking show less, Functions are added to the "about" buttons to go to that section, and then to the E3 icon to return to the main page. Drop-down menu when clicking on the hamburger button is automatically positioned in the header, the home section of the menu closes the "about" section if it is open, the about section opens the same page and the same sponsor section, Flex direction of the past version changed

### CODE REVIEW

**Review by Kamba56 — COMMENTED** [review]
Hi @CarlosZ96,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **has an unprofessional readme and undescriptive PR title.** Kindly customize your readme for your project and add a descriptive PR title that describes the changes added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -241,48 +101,28 @@ Feel free to check the [issues page](../../issues/).
...
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank:<br>
+https://www.microverse.org/ <br>
+https://www.flaticon.com/ <br>
+Freepik1.<br>
+mangsaabguru.<br>
+VectorsMarket.<br>
```
Comment (Lookingfor00) [inline README.md:127]: - [ ] Kindly, make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by Lookingfor00 — CHANGES_REQUESTED** [review]
# Require Change ♻️ 
Hi @CarlosZ96 , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-reach

## Highlights
- Project look awsome 🍏 🌷 
- Code is clean and well organized 🧼 💯 

## Required Changes ♻️
_Check the comments under the review._


- [ ] Please make sure to add a presentation video to showcase your project. Add the link of this presentation video to your README file. Make sure the video meet all the requirement:
```
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.
```

- [ ] Please, make sure to deploy the project on GitHub and add the link to the readme file.

- [ ] Kindly, make sure your project have two HTML file, one for the main page and a other for the about page. Currently your page a loaded dynamically (impressive skill 😮), but the page should be separate file. The navigation between pages also seem to not work and have inspected behavior wash [my loom video for more info](https://www.loom.com/share/13a8c58f7eb549bb992a019e7ca60cf8):
https://www.loom.com/share/13a8c58f7eb549bb992a019e7ca60cf8

- [ ] Please, make sure your main page has only these sections: headline, program, speaker, partner, and footer. And make sure the about page doesn't have partner section and have dark footer

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @CarlosZ96, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Required changes implemented✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Congratulations, Carlitos .... not Alcaraz but Zambrano hahahaha 

A great hug, brother.

Javier.-

**Pull request merged**